service/src/main/java/bio/terra/workspace/service/resource/ValidationUtils.java

```diff
@@ -44,6 +44,10 @@ public static void validateBucketName(String name) {
     }
   }
 
+  public static void validateIpName(String name) {
+    // TODO
```

Comment:
Had trouble finding this originally https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules

I'll implement this shortly...


1-80 | Alphanumerics, underscores, periods, and hyphens.Start with alphanumeric. End alphanumeric or underscore.
-- | --




1-80 | Alphanumerics, underscores, periods, and hyphens.Start with alphanumeric. End alphanumeric or underscore.
-- | --